modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/processors/getAlloyEditorProcessor.js

```diff
@@ -68,6 +69,31 @@ export default function getAlloyEditorProcessor(
 			_editor = AlloyEditor.editable(getEditorWrapper(element), {
 				...editorConfig,
 
+				documentBrowseLinkCallback: (editor, url, changeLinkCallback) => {
+				
+					const itemSelectorDialog = new ItemSelectorDialog({
+						eventName: editor.name + 'selectDocument',
```

Comment:
Wouldn't this be better like `eventName: editorName + 'selectItem',` ?

Reply:
Yes, we can change it to selectItem